pages/compiling/dependencies.rst

```diff
@@ -31,16 +31,15 @@ Fortran compiler is required only if you plan to build the Fortran module.
 
 * Intel compilers (C/C++/Fortran)
 
-  * Known to work with version 14.0.
-  * The 2017 and newer versions of the Intel compiler are **NOT RECOMMENDED** because of a
-    bug in the C compiler, which has been reported to Intel:
-    https://software.intel.com/en-us/forums/intel-c-compiler/topic/684987
+  * Known to work with version 14.0 and Intel OneAPI Toolkits.
+  * The 2017 version of the Intel compiler is **NOT RECOMMENDED** because of a
+    bug in the C compiler.
```

Comment:
Given the lack of testing of anything other than the OneAPI versions, I would suggest simplifying this to just:
```suggestion
  * Known to work with the Intel OneAPI Compilers (version 2022.0.2).
  * Some earlier versions of the Intel compiler (including the 2017 version) are **NOT RECOMMENDED** because of a bug in the C compiler.
```